pkg/collector/corechecks/servicediscovery/module/impl_linux_test.go

```diff
@@ -364,6 +365,78 @@ func buildFakeServer(t *testing.T) string {
 	return filepath.Dir(serverBin)
 }
 
+const (
+	pythonScript = `#! /usr/bin/python
+
+import socket
+import time
+
+HOST = '127.0.0.1'
+PORT = # Empty port
```

Comment:
Is this valid Python?

Reply:
should've been `0` but deleted it accidentally